tests/integration-tests/conftest.py

```diff
@@ -56,6 +56,9 @@ def pytest_addoption(parser):
     parser.addoption("--custom-awsbatchcli-package", help="url to a custom awsbatch cli package")
     parser.addoption("--custom-node-package", help="url to a custom node package")
     parser.addoption("--custom-ami", help="custom AMI to use in the tests")
+    parser.addoption("--vpc-cluster", help="Name of an existing vpc cluster.")
+    parser.addoption("--cluster", help="Use an existing cluster instead of creating one.")
+    parser.addoption("--no-delete", help="Don't delete stacks after tests are complete. Also enables --no-rollback.")
```

Comment:
We can have --no-delete defined as a flag with `action="store_true", default=False,`

Reply:
changed